scripts/setup.sh

```
@@ -0,0 +1,545 @@
+    
+    # Start dev server in background
+    print_info "Starting frontend dev server..."
+    npm run dev > ../var/log/frontend.log 2>&1 &
+    FRONTEND_PID=$!
+    
+    # Save PID
+    echo $FRONTEND_PID > ../var/daemon/pid/frontend.pid
+    
+    # Wait for frontend to be ready
+    print_info "Waiting for frontend to be ready..."
+    for i in {1..30}; do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)